src/components/OrderFilters.vue

```diff
@@ -130,6 +130,11 @@ export default defineComponent({
   },
   methods: {
     async appliedFiltersUpdated(value: string, filterName: string) {
+      const poIds = this.currentOrderFiltersSelected.poIds;
+      if (filterName === 'poIds') {
```

Comment:
I think this should be part of actions, as if are updating the poIds from multiple instance we will have to handle at multiple places, also we may have more such filters

Reply:
Done Sir